apps/ewallet_db/priv/repo/migrations/20180314044304_add_metadata_to_consumptions.exs

```diff
@@ -0,0 +1,18 @@
+defmodule EWalletDB.Repo.Migrations.AddMetadataToConsumptions do
+  use Ecto.Migration
+
+  def change do
+    alter table(:transaction_request_consumption) do
+      add :approved, :boolean, default: false
+      add :finalized_at, :naive_datetime
+      add :expiration_date, :naive_datetime
```

Comment:
**Request:** Maybe add index for `:expiration_date` since it's used by `TransactionConsumption.expire_all/0`?